### PR TITLE
Added nav, hero, social sections for mobile and linter

### PR DESCRIPTION
**Pull request by billionsjoel**
This PR is raised to add the mobile version of my portfolio.

##Tasks
 - added navigation section
 - added hero section
 - added social icons
 - styled for mobile version view
 - added linter for HTML and css

### CODE REVIEW

**Review by julie-ify — CHANGES_REQUESTED** [review]
### Hi @billionsjoel , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [x] You have submitted a project, that does not have a descriptive README. It is recommended to always have a descriptive README that makes our work professional. Please consider adding descriptive README following this [Template](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  <title>Joel Atugonza (Billions)</title>
+  <link rel="stylesheet" href="styles/mobile.css">
+  <link rel="stylesheet" href="styles/style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <nav>
```
Comment (carlos-ssh) [inline index.html]: - The container of the section is a ``header`` tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    <div class="nav-container">
+      <div class="name">Atugonza Joel (Billions)</div>
```
Comment (carlos-ssh) [inline index.html]: - The logo (if the design has one) is on the left, and has a link ``a`` tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        <svg xmlns="http://www.w3.org/2000/svg" width="2.5rem" height="2.5rem" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+          <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+        </svg>
+      </div>
+    </div>
+  </nav>
+  <section id="hero-section">
+    <div class="hero-section-container">
+      <div class="heading">
+        <div class="title">Atugonza Joel</div>
```
Comment (carlos-ssh) [inline index.html]: - The main title (e.g., "I'm Marianne ...") is a ``h1`` tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+        <div class="sub-title">Software Developer</div>
```
Comment (carlos-ssh) [inline index.html]: - The subtitle uses a ``header`` tag ``hx``. 
```
(<h1> <h2> <h3>, etc...)
```

File: index.html [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+      </div>
+    </div>
+  </section>
+
+  <div class="devider">
+  </div>
+
+  <section id="socials">
+    <div class="social-container">
+      <div class="social-icon">
```
Comment (carlos-ssh) [inline index.html]: - The list of icons uses a ``ul`` tag to group them, and ``li`` per icon; each icon has ``a`` link ( ``a`` tag)

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @billionsjoel , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 5 above)
Comment (billionsjoel) [reply index.html:88]: I already did this perfectly.

File: index.html (hunk 3 above)
Comment (billionsjoel) [reply index.html]: Done

File: index.html (hunk 2 above)
Comment (billionsjoel) [reply index.html]: done

File: index.html (hunk 5 above)
Comment (billionsjoel) [reply index.html]: Done

**Review by ShoiraTa — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @billionsjoel ,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
 
- Project is set up correctly ✔️ 
- Descriptive Pull Request ✔️ 
- No linter errors ✔️ 
- Professional documentation ✔️ 

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using

**Pull request merged**